### ISSUE TITLE
Check (e.g. in configure) that Normaliz is version >= 3.0.0

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/NormalizInterface#18]

**Comment by fingolfin** [comment]
For this it wold be nice if there was a `normaliz-config` and/or pkg-config support for normaliz.

(Something similar would be nice to have for GAP, too...)

**Comment by fingolfin** [comment]
Once this is done, we can also remove the source code checks involving `#if NMZ_RELEASE >= 29900`

**Comment by csoeger** [comment]
Adapted the source to require 3.0.0 or newer.
